packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -46,6 +49,8 @@ export default class UpdateConfigure extends EasCommand {
       platform,
     });
 
+    await ensureEASUpdateIsConfiguredInEasJsonAsync(projectDir);
```

Comment:
`update:configure` is supposed to prepare the project for updates, but with this change, you are removing all config steps and only updating eas.json. e.g. You are not checking if expo-updates is installed.

Based on the PR description I would expect that changes in this PR will not affect `update:configure` method and the only difference for the update call would be skipping that eas.json update.

Reply:
I think there may be a misunderstanding here.

This line (52) is in side the `runAsync()` function on the `update:configure` command. When running `update:configure`, we will run two key functions: 
1. `ensureEASUpdateIsConfiguredAsync()`: This installs expo-updates, configures app.json, etc
2. `ensureEASUpdateIsConfiguredInEasJsonAsync()`: This adds channel names to eas.json

Previously, that second function, `ensureEASUpdateIsConfiguredInEasJsonAsync()`, was inside of `ensureEASUpdateIsConfiguredAsync()`, which made it run whenever anyone ran `update`.

This PR removes the eas.json configuration part to only run on `update:configure`, and makes it so that `update` does not run the eas.json configuration part.

When running `update`, we will still install expo-updates + configure app.json etc. Here's where we call that in the `update` command: https://github.com/expo/eas-cli/blob/37bdbe8af17fc750570d6a1cfa54beef42c8c3a8/packages/eas-cli/src/commands/update/index.ts#L203

Reply:
yes, you are right, sorry. I didn't see that both functions were called in configure